functions.photon.php

```diff
@@ -127,10 +127,12 @@ function jetpack_photon_url( $image_url, $args = array(), $scheme = null ) {
 
 	$image_host_path = $image_url_parts['host'] . $image_url_parts['path'];
 
-	// Figure out which CDN subdomain to use
-	srand( crc32( $image_host_path ) );
-	$subdomain = rand( 0, 2 );
-	srand();
+	// Figure out which CDN subdomain to use for this site
+	$subdomain_seed = (int) Jetpack_Options::get_option( 'id' );
```

Comment:
This file is synced on wp.com in mu-plugins, so need to do something WP.com-safe. `Jetpack_Options:get_option( 'id')` will return `false` / `0` for all sites.